packages/teleport/src/Apps/AddApp/Automatically/Automatically.tsx

```diff
@@ -28,27 +28,53 @@ import Validation, { Validator } from 'shared/components/Validation';
 import FieldInput from 'shared/components/FieldInput';
 import { DialogContent, DialogFooter } from 'design/Dialog';
 import { Attempt } from 'shared/hooks/useAttemptNext';
+import cfg from 'teleport/config';
 
 export default function Automatically(props: Props) {
-  const { cmd, onClose, attempt, expires } = props;
+  const { onClose, attempt, expires, token } = props;
 
   const [name, setName] = React.useState('');
   const [uri, setUri] = React.useState('');
+  const [cmd, setCmd] = React.useState('');
+  const [showCmd, setShowCmd] = React.useState(false);
```

Comment:
don't think we need this boolean anymore? can we rely on the falsey/truthy value of `cmd`?